### PR TITLE
Prepare 2.9.0-rc.0 release

### PR DESCRIPTION
**Pull request by brian-brazil**
Signed-off-by: Brian Brazil <brian.brazil@robustperception.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+## 2.9.0-rc.0 / 2019-04-10
+
+* [CHANGE/ENHANCEMENT] update Consul to support catalog.ServiceMultipleTags #5151
```
Comment (brancz) [inline CHANGELOG.md]: nit: we typically start upper case and end with a period

**Comment by brancz** [comment]
lgtm :+1:

**Comment by simonpasquier** [comment]
@krasi-georgiev http://prombench.prometheus.io/5450/prometheus-pr returns a 503 error. Looking at Grafana, it seems like the Prometheus server running the PR has never started.

**Comment by krasi-georgiev** [comment]
yeah the build failed. Not sure why. the build is using the official `make build` command which uses `promu`.  Complains about:
```
scrape/scrape.go:275:11: sp.client.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
```
when I run it locally no problem. I suppose some modules cache problem.


<details><summary>logs</summary>
```
I  >> Creating prometheus binaries
 
I  
 
I  curl -s -L https://github.com/prometheus/promu/releases/download/v0.3.0/promu-0.3.0.linux-amd64.tar.gz | tar -xvzf - -C /tmp/tmp.yt07TjHsj9
 
I  promu-0.3.0.linux-amd64/
 
I  promu-0.3.0.linux-amd64/NOTICE
 
I  promu-0.3.0.linux-amd64/LICENSE
 
I  promu-0.3.0.linux-amd64/promu
 
I  mkdir -p /go/bin
 
I  cp /tmp/tmp.yt07TjHsj9/promu-0.3.0.linux-amd64/promu /go/bin/promu
 
I  rm -r /tmp/tmp.yt07TjHsj9
 
I  >> building binaries
 
I  GO111MODULE=on /go/bin/promu build --prefix /go/src/github.com/prometheus/prometheus
 
I   >   prometheus
 
E  # github.com/prometheus/prometheus/scrape
 
E  scrape/scrape.go:275:11: sp.client.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
 
E  scrape/scrape.go:333:11: oldClient.CloseIdleConnections undefined (type *http.Client has no field or method CloseIdleConnections)
 
E  !! command failed: build -o /go/src/github.com/prometheus/prometheus/prometheus -ldflags -X github.com/prometheus/common/version.Version=2.9.0-rc.0 -X github.com/prometheus/common/version.Revision=69fc906ea4428db2b314b75076d36a301512e49b -X github.com/prometheus/common/version.Branch=pr-branch -X github.com/prometheus/common/version.BuildUser=root@prometheus-test-pr-5450-94b864cc8-v5lnj -X github.com/prometheus/common/version.BuildDate=20190411-10:39:23  -extldflags '-static' -mod=vendor -a -tags netgo github.com/prometheus/prometheus/cmd/prometheus: exit status 2
 
E  make: *** [common-build] Error 1
 
I  Makefile.common:192: recipe for target 'common-build' failed
 
I  ERROR:: Building of binaries failed 
```

</details>

**Comment by simonpasquier** [comment]
You would need to bump the Go version in the builder script: it should be 1.12.

**Comment by krasi-georgiev** [comment]
yeah I just figured that. Thanks Simon.

**Comment by krasi-georgiev** [comment]
all good now and benchmark is running.

**Pull request merged**